src/filters/dates/relative/relativeDates.test.ts

```diff
@@ -0,0 +1,216 @@
+                { dateTime: mockDates(currentDay + 35, currentMonth, currentYear), text: 'go to work', matched: 'in 5 wks' },
+            ],
+        }, {
+            description: 'should parse in a month correctly',
+            case: 'go to work in a month',
+            result: [
+                { dateTime: mockDates(currentDay, currentMonth + 1, currentYear), text: 'go to work', matched: 'in a month' },
+            ],
+        }, {
+            description: 'should parse in X months correctly',
+            case: 'go to work in 5 months',
+            result: [
+                { dateTime: mockDates(currentDay, currentMonth + 5, currentYear), text: 'go to work', matched: 'in 5 months' },
+            ],
+        }, {
+            description: 'should roll over to next year correctly when the year threshold is crossed',
+            case: 'go to work in 12 months',
+            result: [
+                { dateTime: mockDates(currentDay, currentMonth, currentYear + 1), text: 'go to work', matched: 'in 12 months' },
+            ],
+        }, {
+            description: 'should parse next month correctly',
+            case: 'go to work next month',
+            result: [
+                { dateTime: mockNext('month'), text: 'go to work', matched: 'next month' },
+            ],
+        }, {
+            description: 'should parse in a year correctly',
+            case: 'go to work in a year',
+            result: [
+                { dateTime: mockDates(currentDay, currentMonth, currentYear + 1), text: 'go to work', matched: 'in a year' },
+            ],
+        }, {
+            description: 'should parse in X years correctly',
+            case: 'go to work in 5 years',
+            result: [
+                { dateTime: mockDates(currentDay, currentMonth, currentYear + 5), text: 'go to work', matched: 'in 5 years' },
+            ],
+        }, {
+            description: 'should parse next year correctly',
+            case: 'go to work next year',
+            result: [
+                { dateTime: mockNext('year'), text: 'go to work', matched: 'next year' },
+            ],
+        }, {
+            description: 'should parse in a yr correctly',
+            case: 'go to work in a yr',
+            result: [
+                { dateTime: mockDates(currentDay, currentMonth, currentYear + 1), text: 'go to work', matched: 'in a yr' },
+            ],
+        }, {
+            description: 'should parse in X yrs correctly',
+            case: 'go to work in 5 yrs',
+            result: [
+                { dateTime: mockDates(currentDay, currentMonth, currentYear + 5), text: 'go to work', matched: 'in 5 yrs' },
+            ],
+        }];
+
+        testCases.forEach(item => {
+            test(item.description, () => {
+                const parsedText = RelativeDates.parseText(item.case);
+                expect(parsedText).toEqual(item.result);
+            });
+        });
+    });
+
+    describe('getNext', () => {
```

Comment:
I don't think these tests make sense, we are essentially test the `DateTime` library.

Should test by mocking e.g.

```
    static getNext(unit): DateTime {
        return DateTime.utc()
            .plus({ [unit]: 1 })
            .startOf(unit)
            .endOf('day');
    }


    test('getNext()', () => {
        const spy = jest.spyOn(DateTime, 'utc');
         // blah blah blah
        expect(spy).toHaveBeenCalledWith(blah);
    });
```

Reply:
Also why is this a static method? It should be utils function. Or even just stick it outside of the class.

Reply:
> Also why is this a static method? It should be utils function. Or even just stick it outside of the class.

I would prefer to do this as part of a separate PR whereby I will refactor helper methods into proper folder structure.

Reply:
> I think they make sense, we are testing that the correct unit is being passed to DateTime. Regardless, if the library that we rely on fails, we ought to know that as we expose that to the user.

Yeah it makes sense to test, but it doesn't make sense to test the library. If you use lodash you wouldn't test the functions of lodash right?

Reply:
We can leave this for now, just not a good practice imo